package/Resources/Translations/vi/Translation.ini

```diff
@@ -0,0 +1,1691 @@
+[General]
+Name=Tiếng Việt
+Author=Remnant & HuyDepTrai
```

Comment:
Who is a HuyDepTrai? Is that `phatkg191` aka `uzi` aka `mp9` aka about 40k other nicknames man that made Vietnamese translation for RA2?

https://www.moddb.com/games/cc-red-alert-yuris-revenge/downloads/ra2yr-vietnamese-language-pack-unofficial-language-pack